docker-compose.yml

```diff
@@ -13,6 +13,10 @@ services:
     image: hypothesis/elasticsearch:latest
     ports:
       - '127.0.0.1:9200:9200'
+    healthcheck:
+        test: curl --fail --silent http://localhost:9200 >/dev/null
+        interval: 3s
+        start_period: 1m
```

Comment:
Is this necessary? Does h actually fail to start up if Elasticsearch is not running yet? I know `make dev` will actually crash if Postgres isn't up yet, but I'm not sure whether the same is true for Elasticsearch and RabbitMQ. If we can avoid having to wait for Elasticsearch and RabbitMQ then we can keep things simpler (no healthchecks) and save a few seconds on the startup time

Reply:
> Does h actually fail to start up if Elasticsearch is not running yet?

h tries to initialize the Elasticsearch index when it starts, so you get some confusing error messages if it isn't running. Also things will just seem broken if you try to access various pages served by h during the time when Elasticsearch is starting. I think it would be better to direct efforts into speeding up Elasticsearch startup time, as it seems to me that it really ought to be possible for it to start up quicker than it currently does.